core-services/release-controller/_releases/release-ocp-4.13.json

```diff
@@ -198,6 +198,13 @@
         "name": "release-openshift-ocp-installer-e2e-azure-serial-4.13"
       }
     },
+    "azure-sdn-upgrade-micro": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-release-master-ci-4.14-e2e-azure-sdn-upgrade"
+      },
+      "upgrade": true
+    },
```

Comment:
Same here, looks like 4.14 on 4.13 file.

Reply:
This is the one I created and it looks like a copy paste issue on my part.  The priv will get automatically updated when I run the make commands.  I will fix and review.